### ISSUE TITLE
Integrate Mesos Into Already Available Consul Environment?

### ISSUE THREAD

**Issue by akamalov** [CiscoCloud/mesos-consul#17]
This is a question, not an issue:

Is it Possible to integrate Mesos into already available Consul environment?

**Comment by stevendborrelli** [comment]
Yes, they function mostly independently of each other. You can probably use some of the roles at https://github.com/CiscoCloud/microservices-infrastructure to register mesos services in consul, and mesos-consul itself just runs in a docker container.

**Comment by a-nldisr** [comment]
Can you please state the requirements that need to be configured on a running Consul cluster in the documentation.

I have configured a 22 node consul cluster according to the basic consul documentation, added the options based on the playbooks you provided: 
    "bootstrap": 
    "server":
    "datacenter": 
    "advertise_addr": 
    "data_dir": 
    "ui_dir": 
    "encrypt": 
    "verify_incoming":
    "verify_outgoing": 
    "ca_file":
    "cert_file": 
    "key_file":
    "log_level":
    "enable_syslog": 
    "start_join":
    "rejoin_after_leave":
But keep getting connection refused on interfaces that run no firewall. 
I turned off a client on one of the nodes, configured the mesos-consul to use the ssl certificates that the cluster is using and keep getting this error but cannot locate why it refuses to register. More documentation is greatly appreciated.
